tools/php/README.md

```diff
@@ -0,0 +1,27 @@
+# Php bomber
+
+Script for throwing php error to catcher.
+
+## How to run
+
+Prepare `.env` config
+
+    cp .env.sample .env
+
+You can run in two modes: one-time error(once) and non-stop with interval
```

Comment:
You can run the script in two modes: single run (`once`) and periodic run (`interval`)